src/Renderer/Shader/CloudsFS.glsl

```diff
@@ -44,5 +46,9 @@ void main()
 
     gl_FragColor.a = min(time * min( coefDistCam, 1.2) , 1.) * (vUv.y <= 0.75 ? l : (1. - ((vUv.y - 0.75) / 0.25)) * l  );
 
+    if(lightingOn == 1){   // Add lighting
```

Comment:
I'd rather see these types of switches as #if tests, could you update the PR accordingly ?